.travis.yml

```diff
@@ -22,7 +22,7 @@ env:
     - TEST_SPECIFIC_MODULES=presto-mongodb
     - TEST_SPECIFIC_MODULES=presto-redis
     - TEST_SPECIFIC_MODULES=presto-sqlserver,presto-postgresql,presto-mysql
-    - TEST_SPECIFIC_MODULES=presto-kudu
+    - TEST_SPECIFIC_MODULES=presto-kudu MODULES_WITH_EXTRA_TESTS=presto-kudu
```

Comment:
How about `EXTRA_TESTS=true` to avoid duplication. Below can be
```shell
if [[ -v EXTRA_TESTS ]]; then
    ./bin/run_tests_for_modules.sh "${TEST_SPECIFIC_MODULES}"
fi